apps/condo/domains/property/components/panels/Builder/MapConstructor.ts

```diff
@@ -852,7 +852,8 @@ class MapEdit extends MapView {
         const removeSectionIndex = (sectionIndex - 1) >= 0 ? sectionIndex - 1 : sectionIndex
         if (renameNextUnits) this.updateSectionNumbers(removeSectionIndex, renameNextUnits)
 
-        this.editMode = null
```

Comment:
could you please write some tests

Reply:
apps/condo/domains/property/components/panels/Builder/MapConstructor.spec.ts:466
There are tests for this, but for some reason they do not receive this error.
This function is called in them and works correctly when setting `editMode` to `null`, and not in the test, this gives an error of the depth of the call stack